src/models/db/adjustments.js

```diff
@@ -101,8 +101,9 @@ function upsertAdjustment (persistentAdjustment, options) {
   if (options && options.transaction) {
     return _upsertAdjustment(persistentAdjustment, options.transaction)
   } else {
+    // second parameter is number of retries in case database is busy
     return db.withSerializableTransaction((transaction) =>
-      _upsertAdjustment(persistentAdjustment, transaction))
+      _upsertAdjustment(persistentAdjustment, transaction), 5)
```

Comment:
Why 5? Same as above, this kind of constant should be declared at the top of the file